src/main/src/test/java/org/geoserver/feature/retype/RetypingFeatureSourceTest.java

```diff
@@ -72,7 +72,6 @@ public void testSimpleRename() throws IOException {
         SimpleFeatureType target = tb.buildFeatureType();
 
         SimpleFeatureSource retyped = RetypingFeatureSource.getRetypingSource(fs, target);
-        assertTrue(retyped instanceof SimpleFeatureLocking);
```

Comment:
While the new implementation does implement locking, the implementation is slightly different, such that we do not get a SimpleFeatureLocking here, but rather a SimpleFeatureStore.